README.md

```diff
@@ -1,33 +1,30 @@
 # Gameplay
 
-Issue tracker for Unvanquished gameplay related feedback, opinions, proposals
-and new ideas.
+Issue tracker for Unvanquished gameplay related feedback, problems and new ideas.
 
 To have a fruitful discussion, every issue here **must** fall clearly into one
 of the following categories:
 
-## Feedback
+## Experience & Feedback
```

Comment:
Maybe even just "Experience"? "Feedback" is indeed a vague term and having both here doesn't really improve upon this.